tests/percy/photographer.js

```diff
@@ -49,6 +49,9 @@ class Photographer {
 
     await this._pageNavigator.gotoVerticalPage('events',{ query: 'vrginia' });
     await this._camera.snapshot('vertical-search--spellcheck');
+
+    await this._pageNavigator.gotoVerticalPage('financial_professionals',{ query: 'connor' });
```

Comment:
nit but space!

Reply:
oops, I accidentally merged this PR into the feature branch. I reverted that and put up a new pr here: https://github.com/yext/answers-hitchhiker-theme/pull/815